web-test-runner.utils.js

```diff
@@ -157,14 +157,14 @@ const vrtHTML =
     </html>`;
 
 export let vrtGroups = [];
-const themeVariants = ['spectrum', 'express', 'spectrum-two'];
+const systemVariants = ['spectrum', 'express', 'spectrum-two'];
 const colors = ['lightest', 'light', 'dark', 'darkest'];
 const scales = ['medium', 'large'];
 const directions = ['ltr', 'rtl'];
-themeVariants.forEach((themeVariant) => {
+systemVariants.forEach((systemVariant) => {
     colors.forEach((color) => {
         if (
-            themeVariant === 'spectrum-two' &&
+            systemVariant === 'spectrum-two' &&
             (color === 'lightest' || color === 'darkest')
```

Comment:
Reference here too!

Reply:
Keeping lightest and darkest in our test suites.